packages/playground/src/components/Playground.tsx

```diff
@@ -23,7 +23,7 @@ export default function Playground({ themes, validators }: PlaygroundProps) {
   const [formData, setFormData] = useState<any>(samples.Simple.formData);
   const [extraErrors, setExtraErrors] = useState<ErrorSchema | undefined>();
   const [shareURL, setShareURL] = useState<string | null>(null);
-  const [theme, setTheme] = useState<string>('default');
+  const [theme, setTheme] = useState<string>('carbon');
```

Comment:
Why make this change?

Reply:
For dev convience. I'll add a TODO and remove before merge.